extensions/amp-analytics/0.1/vendors.js

```diff
@@ -438,6 +438,35 @@ export const ANALYTICS_CONFIG = {
         '${actionParameter}${parameterSuffix}',
     },
   },
+
+  'shinystat': {
+    'requests': {
+      'ssurl': 'https://amp.shinystat.com/cgi-bin/getcod.cgi',
+      'ssparam': 'AMP=1' +
+                 '&USER=${account}' +
+                 '&AMPPAG=${page}' +
+                 '&AMPHR=${canonicalUrl}' +
+                 '&AMPREFER=${documentReferrer}' +
+                 '&AMPRES=${screenWidth}X${screenHeight}' +
+                 '&AMPCOLOR=${screenColorDepth}' +
+                 '&AMPCID=${clientId(AMP_CID)}' +
+                 '&AMPPID=${pageViewId}' +
+                 '&AMPTTL=${title}' +
+                 '&AMPRQC=${requestCount}' +
+                 '&AMPVIE=${viewer}' +
+                 '&AMPSSIDC=${queryParam(ssidc)}' +
+                 '&${ampp1}=${ampv1}' +
+                 '&${ampp2}=${ampv2}',
+      'pageview': '${ssurl}?${ssparam}',
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframePing': true,
```

Comment:
On Wed, Mar 30, 2016 at 1:41 AM, ShinyStat <notifications@github.com> wrote:

> In extensions/amp-analytics/0.1/vendors.js
> <https://github.com/ampproject/amphtml/pull/2556#discussion_r57854144>:
>
> > +                 '&AMPCOLOR=${screenColorDepth}' +
> > +                 '&AMPCID=${clientId(AMP_CID)}' +
> > +                 '&AMPPID=${pageViewId}' +
> > +                 '&AMPTTL=${title}' +
> > +                 '&AMPRQC=${requestCount}' +
> > +                 '&AMPVIE=${viewer}' +
> > +                 '&AMPSSIDC=${queryParam(ssidc)}' +
> > +                 '&${ampp1}=${ampv1}' +
> > +                 '&${ampp2}=${ampv2}',
> > +      'pageview': '${ssurl}?${ssparam}',
> > +    },
> > +    'triggers': {
> > +      'pageview': {
> > +        'on': 'visible',
> > +        'request': 'pageview',
> > +        'iframePing': true,
>
> My company signed the CLA. I quote my one week old answer:
>
> Thanks for your help. We tested it successfully. Safari allows the
> javascript code to access session and local storage data. Even though it
> deletes everything on exit, data persist long enough to collect the
> informations we need.
>
> Is there any other issue?
>
I haven't seen a response/update on
https://github.com/ampproject/amphtml/pull/2556#issuecomment-197465102  did
that offline conversation happen? what was the result of that?


> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amphtml/pull/2556/files/21540b3da5d24ede1b30dd584786a5929ee3a8fe#r57854144>
>